xsd-parser-rs/src/parser/tests.rs

```diff
@@ -63,4 +63,65 @@ mod test {
             _ => unreachable!(),
         }
     }
+
+    #[test]
```

Comment:
Is it possible to make an integration test under `xsd-parser-rs/src/tests` to see what's going to be generated in this case?

Reply:
Done